### ISSUE TITLE
customizing link style with any color sets `fill` property to `black` instead of `none`

### ISSUE THREAD

**Issue by g3o2** [knsv/mermaid#416]
example code

```markdown
graph TB;
    A-->B;
    linkStyle 0 stroke:red;
    B-->C;
    C-->A;
```
http://knsv.github.io/mermaid/live_editor/#/edit/Z3JhcGggVEI7CiAgICBBLS0-QjsKICAgIGxpbmtTdHlsZSAwIHN0cm9rZTpyZWQ7CiAgICBCLS0-QzsKICAgIEMtLT5BOw

**Comment by knsv** [comment]
I agree! Fill noen definitely looks better... :) This should be fixed.

**Comment by joshuacolvin** [comment]
I can help with this.

**Comment by knsv** [comment]
That is appreciated!

**Comment by knsv** [comment]
Closed by release 7.0.0, please reopen if issues remain.
